.github/scripts/hermetic_library_generation.sh

```diff
@@ -107,7 +104,6 @@ docker run \
   -u "$(id -u):$(id -g)" \
   -v "$(pwd):${workspace_name}" \
   -v "$HOME"/.m2:/home/.m2 \
-  -e GENERATOR_VERSION="${generator_version}" \
```

Comment:
I think we can remove the `.m2` volume above also?

Reply:
Done